### DISCUSSION TITLE
Could we have a better icon library please?

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13360] [Feature Requests]
The material icons are pretty limited in their scope - e.g. an icon for anything related to banking? Well, there's a food bank icon! The usual font library is font-awesome which has just released version 6 and have some 16,000 icons: https://fontawesome.com/search?p=1 and you can find icons that are more representative of nouns.

**Comment by JonathanSchndr** [comment]
i also think https://heroicons.com/ is very good!

**Reply by sxsws** [reply]
only 230 icons though :(

**Reply by JonathanSchndr** [reply]
yes :( i think the free solid fontawesome icons are pretty ugly :sweat_smile:

**Reply by benhaynes** [reply]
I think it's a bit of quantity over quality with FontAwesome. Material Design has 6K+ icons, and [has several for bank](https://fonts.google.com/icons?icon.query=bank). The name isn't always exact, so you have to find representational icons... but the icons are consistent and clean, with a variable weight and solid/outline.

I'm all about adding support for custom icon sets, but (for now) the default set will likely remain MD.

**Reply by rijkvanzanten** [reply]
Also, I don't think we can legally use v6 with the full 16k+ icon set as per their licensing and plans: https://fontawesome.com/plans

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
